app/src/main/java/com/padc/cooking/activities/MainActivity.java

```diff
@@ -28,12 +71,40 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        if (savedInstanceState == null){
+            HomeFragment homeFragment = new HomeFragment();
```

Comment:
Fragment creation should be with *static factory pattern* even if there is no parameter yet.